tests/instances.tcl

```diff
@@ -298,6 +299,8 @@ proc parse_options {} {
             set val2 [lindex $::argv [expr $j+2]]
             dict set ::global_config $val $val2
             incr j 2
+        } elseif {$opt eq {--fastfail}} {
```

Comment:
Seem to like `--exit-on-failure` over `fastfail`.

Reply:
@hpatro `exit-on-failure` is just a little bit longer as an option, I believe that's why we have `-stop` stands for `stop-on-failure`

Reply:
I think fast fail encompasses the error. It's also not like I can ever correctly guess the flags for our TCL tests. The number of times I have to look up "--dont-clean" is very high. Also by convention shouldn't it be `--fast-fail`, since we seem to always break up words with dashes.

Reply:
thx for your review, `--fast-fail` looks good to me